js/src/browser/bytes.js

```diff
@@ -57,7 +57,10 @@ export function copy(source: Uint8Array, target: Uint8Array, targetStart: number
 }
 
 export function slice(buff: Uint8Array, start: number, end: number): Uint8Array {
-  return buff.slice(start, end);
+  // Safari does not have slice on typed arrays.
```

Comment:
But now that I look at the spec we can use `ArrayBuffer.prototype.slice` which does work in Safari 9.